lambdas/issuecredential/src/test/java/uk/gov/di/ipv/cri/address/api/handler/IssueCredentialHandlerTest.java

```diff
@@ -277,6 +276,11 @@ void shouldThrowAWSExceptionWhenAServerErrorOccursDuringRetrievingAnAddressItemW
     }
 
     private void setupEventProbeErrorBehaviour() {
+        setupEventProbeBehaviour();
```

Comment:
To adhere to the single responsibility principle it would make sense to undo the amend to the `setupEventProbeErrorBehaviour` method
& 
Then the `setupEventProbeBehaviour()` method would simply contain the following line:
when(mockEventProbe.log(any(Level.class), any(String.class))).thenReturn(mockEventProbe);
If possible then change the above line to: 
when(mockEventProbe.log(Level.INFO, "found session")).thenReturn(mockEventProbe);

The use of `any` should generally be avoided where possible

The test methods can then simply invoke whichever setup* methods they require